### ISSUE TITLE
Ubuntu hangs on Exiting boot services

### ISSUE THREAD

**Issue by Delamcode** [utmapp/UTM#4499]
**Describe the issue**
Stuck in Start Boot Option; EFI stub: Exiting boot services...

**Configuration (required)**
* UTM Version: 4.0.7
* OS Version: 15.5 b4
* Device Model: iPad Air 5
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? TrollStore 

**Crash log**
N/a

**Debug log**
Check replies 

**Upload VM**
Check replies

**Comment by Delamcode** [comment]
[debug.log](https://github.com/utmapp/UTM/files/9759375/debug.log)

**Comment by ctnadovich** [comment]
Yep. I see this too. On an MacBook Air M2 processor. I had full disk encryption selected in the installer, if that matters.

**Comment by Delamcode** [comment]
I doubt it's the encryption, cause I had it off

**Comment by ctnadovich** [comment]
If I turn encryption off, the install works fine. So this may be a clue. Perhaps something is being prompted for and the prompt never gets wired to the console. In my case it's the volume password. In your case, something else. Just a guess.
